packages/webapp/src/pages/Logs/Search.tsx

```diff
@@ -87,13 +161,19 @@ export const LogsSearch: React.FC = () => {
             </div>
 
             <div className="flex gap-2 justify-between">
-                <div className="w-full">
-                    <Input before={<MagnifyingGlassIcon className="w-5 h-5" />} placeholder="Search operations..." />
-                </div>
+                <div className="w-full">{/* <Input before={<MagnifyingGlassIcon className="w-5 h-5" />} placeholder="Search operations..." /> */}</div>
                 <MultiSelect label="Status" options={statusOptions} selected={states} defaultSelect={statusDefaultOptions} onChange={setStates} all />
+                <TypesSelect selected={types} onChange={setTypes} />
+                <SearchableMultiSelect label="Integration" selected={integrations} category={'config'} onChange={setIntegrations} />
+                <SearchableMultiSelect label="Connection" selected={connections} category={'connection'} onChange={setConnections} />
+                <SearchableMultiSelect label="Script" selected={syncs} category={'syncConfig'} onChange={setSyncs} />
+                <Button variant="zombieGray" size={'xs'}>
+                    <LightningBoltIcon />
+                    Live
+                </Button>
```

Comment:
This button is doing nothing for now. Is that on purpose?

Reply:
yep, there will be a calendar there